mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -14,6 +14,14 @@ object English : LocalizedStrings {
   override val authenticateSwitchToLogIn = "Log in".uppercase()
   override val authenticateSwitchToRegister = "Sign up".uppercase()
   override val authenticateErrorFailure = "Something went wrong when authenticating"
+  override val authenticationBadPasswordFailure =
+      "Your password should be at least 6 characters long, and contain at least one number, capital letter and special character"
```

Comment:
Are you sure about that ? I see some valid password on some tests that don't have these requirements

Reply:
It actually doesn’t support the “Should contain at least one number, capital letter and special character as I realized the Firebase Error does not support that. But I can add that manually if we want to support it.